scipy/interpolate/tests/test_bsplines.py

```diff
@@ -2255,6 +2257,27 @@ def test_weighted_smoothing_spline(self):
                                  f' points than the original one: {orig:.4} < '
                                  f'{weighted:.4}')
 
+    @pytest.mark.parametrize("lam", [1.0, None])
+    @pytest.mark.parametrize("axis", range(1, 4))
+    def test_shapes_axis(self, axis, lam):
```

Comment:
Is there anything covered here that is not covered by `TestBatch`? (Maybe just the use of `make_smoothing_spline` without an explicit `axis`?) 

If they are redundant, we could consider removing this and the tests added in the other recent PR.